homepage/tests/routing_tests.py

```diff
@@ -5,7 +5,7 @@
     ('', 200),
     ('/courses/', 200),
     ('/reviews/', 200),
-    ('/add_review/', 200)
+    ('/add_review/10231', 200)

```

Comment:
Referring to a specific course in this file is a bit out of context IMO, you have the same test in post_review_tests (covering all the tests data courses). I think it's better to remove it from here since it's already covered.

Reply:
@ellaml Removed